activerecord/lib/active_record/associations.rb

```diff
@@ -300,10 +300,10 @@ def association_instance_set(name, association)
     #
     # === A word of warning
     #
-    # Don't create associations that have the same name as instance methods of
-    # ActiveRecord::Base. Since the association adds a method with that name to
-    # its model, it will override the inherited method and break things.
-    # For instance, +attributes+ and +connection+ would be bad choices for association names.
+    # Don't create associations that have the same name as [instance methods](http://apidock.com/rails/ActiveRecord/Base) of
```

Comment:
The official Rails code page for AR::Base would be: http://api.rubyonrails.org/classes/ActiveRecord/Base.html